packages/atomic/storybookUtils/search/search-interface-wrapper.tsx

```diff
@@ -40,3 +40,17 @@ export const wrapInSearchInterface = (
     });
   },
 });
+
+export const playExecuteFirstSearch: (
+  context: StoryContext
+) => Promise<void> = async ({canvasElement, step}) => {
+  const canvas = within(canvasElement);
+
+  const searchInterface =
+    await canvas.findByTestId<HTMLAtomicSearchInterfaceElement>(
+      'root-interface'
+    );
+  await step('Execute the first search', async () => {
+    await searchInterface!.executeFirstSearch();
+  });
+};
```

Comment:
Same as the other comment, I dont understand the need for this. The exact same code is already executed in `play()`

Reply:
See above